lib/order.rb

```diff
@@ -0,0 +1,30 @@
+require_relative 'checkout'
+
+class Order
```

Comment:
Excellent! You read / wrote to instance variables via the class' public interface.